### ISSUE TITLE
Helm chart distribution

### ISSUE THREAD

**Issue by txomon** [GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp#131]
### TL;DR

Following customs on kubernetes resource distribution, it would be nice to have a chart that could be used to install the provider, similar to https://github.com/kubernetes-sigs/secrets-store-csi-driver/tree/master/charts/secrets-store-csi-driver#configuration

### Design

**Proposal**
Create a helm chart that either has the official secrets store csi drive helm chart as a dependency, or an independent one that the user needs to coordinate installation for.

**Resources**

- Offical secrets store csi driver helm chart https://github.com/kubernetes-sigs/secrets-store-csi-driver/tree/master/charts/secrets-store-csi-driver#configuration

**Comment by red8888** [comment]
Has the chart not been published yet? What is the right URL to pass `helm repo add`?

**Comment by dannysauer** [comment]
https://github.com/helm/chart-releaser-action would be an easy way to actually publish the chart.  I can't do the publishing part and generating a repo secret in a PR, or I'd open a PR for this and save the project some work. :D

CC: #192
